Dockerfile

```diff
@@ -28,41 +28,87 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN curl -fsSL https://deb.nodesource.com/setup_14.x | bash - \
  && apt-get install -y nodejs
 
-# Downloading dependencies, these should be pinned to specific versions.
+# Add dependencies. All should be pinned to specific versions, except
+# Nextstrain-maintained software.
 # This includes pathogen-specific workflow dependencies. Since we only maintain a
 # single Docker image to support all pathogen workflows, some pathogen-specific
 # functionality must live in this Dockerfile. The following dependencies may be
 # used by multiple pathogen workflows, but they have been commented according to
 # the original pathogen that added these dependencies.
 
-# mafft
-WORKDIR /build/mafft
-RUN curl -fsSL https://mafft.cbrc.jp/alignment/software/mafft-7.475-linux.tgz \
-  | tar xzvpf - --no-same-owner --strip-components=2 mafft-linux64/mafftdir/
+# Create directories to be copied in final stage.
+RUN mkdir -p /out/bin /out/share /out/libexec
```

Comment:
Consider naming this `/final` instead of `/out` to mark that it's for the `final` stage?

Reply:
I like that. Done: cf4fdce6a283f56665e256b46fb8e18e8c135a68